test_runner/src/main/kotlin/ftl/json/SavedMatrix.kt

```diff
@@ -28,7 +28,9 @@ data class SavedMatrix(
     val state: String = "",
     val gcsPath: String = "",
     val webLink: String = "",
-    val downloaded: Boolean = false,
+    // this is variable intentionally, this is workaround to pass changes while fetching artifacts
+    // todo think about different solution
+    var downloaded: Boolean = false,
```

Comment:
Changed to `var`. The old implementation had a bug and none of `SavedMatrix` had `downloaded` updated during artifact fetch. This is a quick fix for this issue. We should think about a different solution but in a separate issue. 

Might be fixed within another refactor task (?)

@jan-gogo